src/process/channel.rs

```diff
@@ -107,6 +109,18 @@ impl Channel {
         }
     }
 
+    pub fn close_receiver(&self) -> Result<()> {
+        unistd::close(self.receiver.as_raw_fd())?;
+
+        Ok(())
+    }
+
+    pub fn close_sender(&self) -> Result<()> {
+        unistd::close(self.sender.as_raw_fd())?;
+
+        Ok(())
```

Comment:
How about this code?
```suggestion
        unistd::close(self.receiver.as_raw_fd())
    }

    pub fn close_sender(&self) -> Result<()> {
        unistd::close(self.sender.as_raw_fd())
```

Reply:
unistd functions will return a different error type than anyhow result, so we have to use `?` for a conversion.